src/com/codeenginestudio/bookManagement/validator/BookValidator.java

```diff
@@ -0,0 +1,37 @@
+package com.codeenginestudio.bookManagement.validator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BookValidator {
+
+	public static List<Map<String, String>> _validate(String bookName) {
```

Comment:
1. `validate` is better!
2. As I mentioned above, Map is enough for this functional!